tests/runtest.sh

```diff
@@ -429,22 +426,26 @@ declare -a playlistTests
 
 function load_unsupported_tests {
     # Load the list of tests that are not supported on this platform. These tests are disabled (skipped) permanently.
-    # 'readarray' is not used here, as it includes the trailing linefeed in lines placed in the array.
+    # bash in Mac OS X doesn't support 'readarray', so sing alternate way instead.
+    # readarray -t unsupportedTests <"$(dirname "$0")/testsUnsupportedOutsideWindows.txt"
```

Comment:
This comment (at all places) doesn't seem to add much value (other than preventing someone to change the loop to readarray in the future, but the CI would quickly show it doesn't work on OSX), so I would recommend removing it.
Also, since you are modifying the code, would you mind also extracting the file content reading into a separate function with the .txt file name as a parameter and using it at all four places instead of repeating the same loop in those places?